packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -72,8 +188,9 @@ This package contains documentation for rubygem-%{gem_name}.
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -a .%{gem_dir}/* \
+cp -pa .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
+rm %{buildroot}/%{gem_instdir}/.babelrc
```

Comment:
I actually needed to remove it, as the build was failing for some weird reasons: it looks like we don't need any of those dependencies for the production builds: the new UI with the rpms I've build locally worked just fine without the file